data/locations.yml

```diff
@@ -8,12 +8,16 @@
     Southampton
     Hampshire
     SO17 1BJ
-"building-59":
-  link: http://data.southampton.ac.uk/building/59.html
-  name: Building 59
-  detail: University of Southampton’s recently refurbished electronics teaching labs
+cube:
+  link: http://data.southampton.ac.uk/point-of-service/cube.html
+  name: The Cube
+  detail: University of Southampton’s Highfield Campus
   address: |
     University Road
     Southampton
     Hampshire
     SO17 1BJ
+"building-59":
+  link: http://data.southampton.ac.uk/building/59.html
+  name: Building 59
+  detail: University of Southampton’s recently refurbished electronics teaching labs
```

Comment:
Building 59 appears to have lost its address, possibly something went wrong in a merge?